refactor(account-management)_: improvements on package and type naming and introducing a readme file

saledjenic opened the pull request:
This PR finalizes the account management refactor (the first phase). Brings a better name for the accounts package, better name for the old `DefaultManager` type and introduces a README for the `accountsmanagement` package.

The next phase should be about adding account persistence to `accountsmanagement` package, after which this package will be fully autonomous.

----
Reference to other accounts-management refactor PRs:
- https://github.com/status-im/status-go/pull/6605
- https://github.com/status-im/status-go/pull/6612
- https://github.com/status-im/status-go/pull/6640
- https://github.com/status-im/status-go/pull/6660
- https://github.com/status-im/status-go/pull/6668
- https://github.com/status-im/status-go/pull/6673
- https://github.com/status-im/status-go/pull/6686
- https://github.com/status-im/status-go/pull/6687
- https://github.com/status-im/status-go/pull/6692

saledjenic committed:
refactor(account-management)_: account package renamed to accountsmanagement package

saledjenic committed:
refactor(account-management)_: rename DefaultManager to AccountsManager and update references

saledjenic committed:
chore(accounts-management)_: add README for Accounts Management package

Added a detailed README.md file outlining the functionality, structure, key components, and usage examples for the Accounts Management package.

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (88)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 501615d8 | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6687/1/) | 2025-06-17 07:45:27 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250617-074238-501615-pr6687.aar) |
| :heavy_check_mark: | 501615d8 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6687/1/) | 2025-06-17 07:45:39 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250617-074243-501615-pr6687.zip) |
| :heavy_check_mark: | 501615d8 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6687/1/) | 2025-06-17 07:46:21 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250617-074240-501615-pr6687.zip) |
| :heavy_check_mark: | 501615d8 | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6687/1/) | 2025-06-17 07:47:07 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250617-074238-501615-pr6687.zip) |
| :heavy_check_mark: | 501615d8 | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6687/1/) | 2025-06-17 07:47:42 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250617-074250-501615-pr6687.zip) |
| :heavy_check_mark: | 501615d8 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6687/1/) | 2025-06-17 07:48:25 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250617-074245-501615-pr6687.zip) |
| :heavy_multiplication_x: | 501615d8 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/1/) | 2025-06-17 07:50:33 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/1//consoleText) |
| :heavy_check_mark: | 501615d8 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6687/1/) | 2025-06-17 07:53:32 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250617-074240-501615-pr6687.zip) |
| :heavy_multiplication_x: | 501615d8 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/1/) | 2025-06-17 08:07:06 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/1//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 79f565bf | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6687/2/) | 2025-06-17 08:12:24 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250617-080934-79f565-pr6687.zip) |
| :heavy_check_mark: | 79f565bf | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6687/2/) | 2025-06-17 08:12:38 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250617-080928-79f565-pr6687.aar) |
| :heavy_check_mark: | 79f565bf | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6687/2/) | 2025-06-17 08:13:13 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250617-080931-79f565-pr6687.zip) |
| :heavy_check_mark: | 79f565bf | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6687/2/) | 2025-06-17 08:13:55 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250617-080930-79f565-pr6687.zip) |
| :heavy_check_mark: | 79f565bf | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6687/2/) | 2025-06-17 08:14:17 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250617-080940-79f565-pr6687.zip) |
| :heavy_check_mark: | 79f565bf | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6687/2/) | 2025-06-17 08:15:11 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250617-080937-79f565-pr6687.zip) |
| :heavy_multiplication_x: | 79f565bf | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/2/) | 2025-06-17 08:16:13 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/2//consoleText) |
| :heavy_check_mark: | 79f565bf | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6687/2/) | 2025-06-17 08:18:20 | ~8 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250617-080932-79f565-pr6687.zip) |
| :heavy_check_mark: | 79f565bf | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/3/) | 2025-06-17 08:31:38 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/3//consoleText) |
| :heavy_multiplication_x: | 79f565bf | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/2/) | 2025-06-17 08:34:08 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/2//consoleText) |
| | | | | | | |
| :heavy_check_mark: | b9d3a9d0 | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6687/3/) | 2025-06-17 10:05:15 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250617-100039-b9d3a9-pr6687.zip) |
| :heavy_check_mark: | b9d3a9d0 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6687/3/) | 2025-06-17 10:05:49 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250617-100035-b9d3a9-pr6687.zip) |
| :heavy_check_mark: | b9d3a9d0 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6687/3/) | 2025-06-17 10:06:49 | ~6 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250617-100033-b9d3a9-pr6687.zip) |
| :heavy_check_mark: | b9d3a9d0 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6687/3/) | 2025-06-17 10:07:13 | ~6 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250617-100027-b9d3a9-pr6687.zip) |
| :heavy_check_mark: | b9d3a9d0 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6687/3/) | 2025-06-17 10:14:31 | ~14 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250617-100026-b9d3a9-pr6687.aar) |
| :heavy_check_mark: | b9d3a9d0 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6687/3/) | 2025-06-17 10:16:06 | ~15 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250617-100028-b9d3a9-pr6687.zip) |
| :heavy_check_mark: | b9d3a9d0 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/4/) | 2025-06-17 10:24:04 | ~23 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/4//consoleText) |
| :heavy_check_mark: | b9d3a9d0 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6687/3/) | 2025-06-17 10:25:45 | ~25 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250617-100030-b9d3a9-pr6687.zip) |
| :heavy_multiplication_x: | b9d3a9d0 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/3/) | 2025-06-17 10:42:48 | ~42 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | c036f7c8 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6687/4/) | 2025-06-17 11:32:39 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250617-112819-c036f7-pr6687.zip) |
| :heavy_check_mark: | c036f7c8 | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6687/4/) | 2025-06-17 11:33:10 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250617-112823-c036f7-pr6687.zip) |
| :heavy_check_mark: | c036f7c8 | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6687/4/) | 2025-06-17 11:33:17 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250617-112811-c036f7-pr6687.zip) |
| :heavy_check_mark: | c036f7c8 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6687/4/) | 2025-06-17 11:33:53 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250617-112816-c036f7-pr6687.zip) |
| :heavy_check_mark: | c036f7c8 | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6687/4/) | 2025-06-17 11:34:38 | ~6 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250617-112811-c036f7-pr6687.aar) |
| :heavy_check_mark: | c036f7c8 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6687/4/) | 2025-06-17 11:36:28 | ~8 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250617-112813-c036f7-pr6687.zip) |
| :heavy_multiplication_x: | c036f7c8 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/5/) | 2025-06-17 11:43:03 | ~14 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/5//consoleText) |
| :heavy_check_mark: | c036f7c8 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6687/4/) | 2025-06-17 11:44:16 | ~16 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250617-112814-c036f7-pr6687.zip) |
| :heavy_multiplication_x: | c036f7c8 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/4/) | 2025-06-17 12:01:31 | ~33 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | c82c1121 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6687/5/) | 2025-06-17 12:22:34 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250617-121707-c82c11-pr6687.zip) |
| :heavy_check_mark: | c82c1121 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6687/5/) | 2025-06-17 12:23:13 | ~6 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250617-121704-c82c11-pr6687.zip) |
| :heavy_check_mark: | c82c1121 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6687/5/) | 2025-06-17 12:23:25 | ~6 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250617-121659-c82c11-pr6687.zip) |
| :heavy_check_mark: | c82c1121 | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6687/5/) | 2025-06-17 12:24:39 | ~7 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250617-121711-c82c11-pr6687.zip) |
| :heavy_check_mark: | c82c1121 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6687/5/) | 2025-06-17 12:26:02 | ~9 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250617-121657-c82c11-pr6687.aar) |
| :heavy_check_mark: | c82c1121 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6687/5/) | 2025-06-17 12:26:46 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250617-121700-c82c11-pr6687.zip) |
| :heavy_check_mark: | c82c1121 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6687/5/) | 2025-06-17 12:34:21 | ~17 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250617-121701-c82c11-pr6687.zip) |
| :heavy_check_mark: | c82c1121 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/6/) | 2025-06-17 12:34:38 | ~17 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/6//consoleText) |
| :heavy_multiplication_x: | c82c1121 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/5/) | 2025-06-17 12:52:38 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/5//consoleText) |
| :heavy_multiplication_x: | c82c1121 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/6/) | 2025-06-18 06:29:05 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/6//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 66f38658 | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6687/6/) | 2025-06-18 07:05:17 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250618-070226-66f386-pr6687.zip) |
| :heavy_check_mark: | 66f38658 | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6687/6/) | 2025-06-18 07:05:27 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250618-070220-66f386-pr6687.aar) |
| :heavy_check_mark: | 66f38658 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6687/6/) | 2025-06-18 07:06:08 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250618-070222-66f386-pr6687.zip) |
| :heavy_check_mark: | 66f38658 | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6687/6/) | 2025-06-18 07:06:56 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250618-070221-66f386-pr6687.zip) |
| :heavy_check_mark: | 66f38658 | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6687/6/) | 2025-06-18 07:07:53 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250618-070228-66f386-pr6687.zip) |
| :heavy_multiplication_x: | 66f38658 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/7/) | 2025-06-18 07:08:47 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/7//consoleText) |
| :heavy_check_mark: | 66f38658 | [#6](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6687/6/) | 2025-06-18 07:10:00 | ~7 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250618-070232-66f386-pr6687.zip) |
| :heavy_check_mark: | 66f38658 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6687/6/) | 2025-06-18 07:13:27 | ~11 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250618-070224-66f386-pr6687.zip) |
| :heavy_multiplication_x: | 66f38658 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/7/) | 2025-06-18 07:26:42 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/7//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 18fbbab6 | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6687/7/) | 2025-06-18 07:24:48 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250618-072151-18fbba-pr6687.aar) |
| :heavy_check_mark: | 18fbbab6 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6687/7/) | 2025-06-18 07:24:56 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250618-072156-18fbba-pr6687.zip) |
| :heavy_check_mark: | 18fbbab6 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6687/7/) | 2025-06-18 07:25:32 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250618-072154-18fbba-pr6687.zip) |
| :heavy_check_mark: | 18fbbab6 | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6687/7/) | 2025-06-18 07:26:15 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250618-072152-18fbba-pr6687.zip) |
| :heavy_check_mark: | 18fbbab6 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6687/7/) | 2025-06-18 07:27:29 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250618-072159-18fbba-pr6687.zip) |
| :heavy_check_mark: | 18fbbab6 | [#7](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6687/7/) | 2025-06-18 07:28:04 | ~6 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250618-072203-18fbba-pr6687.zip) |
| :heavy_multiplication_x: | 18fbbab6 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/8/) | 2025-06-18 07:29:08 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/8//consoleText) |
| :heavy_check_mark: | 18fbbab6 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6687/7/) | 2025-06-18 07:31:17 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250618-072154-18fbba-pr6687.zip) |
| :heavy_multiplication_x: | 18fbbab6 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/8/) | 2025-06-18 07:51:25 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/8//consoleText) |
| :heavy_check_mark: | 18fbbab6 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/9/) | 2025-06-18 10:03:31 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/9//consoleText) |
| :heavy_multiplication_x: | 18fbbab6 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/9/) | 2025-06-18 10:21:05 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/9//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 11913154 | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6687/8/) | 2025-06-19 13:30:03 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250619-132712-119131-pr6687.aar) |
| :heavy_check_mark: | 11913154 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6687/8/) | 2025-06-19 13:30:11 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250619-132718-119131-pr6687.zip) |
| :heavy_check_mark: | 11913154 | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6687/8/) | 2025-06-19 13:30:49 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250619-132714-119131-pr6687.zip) |
| :heavy_check_mark: | 11913154 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6687/8/) | 2025-06-19 13:31:08 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250619-132721-119131-pr6687.zip) |
| :heavy_check_mark: | 11913154 | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6687/8/) | 2025-06-19 13:31:34 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250619-132713-119131-pr6687.zip) |
| :heavy_check_mark: | 11913154 | [#8](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6687/8/) | 2025-06-19 13:31:55 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250619-132724-119131-pr6687.zip) |
| :heavy_check_mark: | 11913154 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/10/) | 2025-06-19 13:35:08 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/10//consoleText) |
| :heavy_check_mark: | 11913154 | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6687/8/) | 2025-06-19 13:39:52 | ~12 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250619-132716-119131-pr6687.zip) |
| :heavy_multiplication_x: | 11913154 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/10/) | 2025-06-19 13:53:39 | ~26 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/10//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 4811c206 | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6687/9/) | 2025-06-19 13:56:17 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250619-135226-4811c2-pr6687.zip) |
| :heavy_check_mark: | 4811c206 | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6687/9/) | 2025-06-19 13:56:17 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250619-135228-4811c2-pr6687.zip) |
| :heavy_check_mark: | 4811c206 | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6687/9/) | 2025-06-19 13:56:49 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250619-135220-4811c2-pr6687.aar) |
| :heavy_check_mark: | 4811c206 | [#9](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6687/9/) | 2025-06-19 13:57:30 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250619-135232-4811c2-pr6687.zip) |
| :heavy_check_mark: | 4811c206 | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6687/9/) | 2025-06-19 13:58:42 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250619-135222-4811c2-pr6687.zip) |
| :heavy_check_mark: | 4811c206 | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6687/9/) | 2025-06-19 13:59:18 | ~6 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250619-135221-4811c2-pr6687.zip) |
| :heavy_check_mark: | 4811c206 | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6687/9/) | 2025-06-19 14:05:22 | ~12 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250619-135224-4811c2-pr6687.zip) |
| :heavy_multiplication_x: | 4811c206 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/11/) | 2025-06-19 14:08:09 | ~15 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/11//consoleText) |
| :heavy_check_mark: | 4811c206 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/11/) | 2025-06-19 14:28:24 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/11//consoleText) |
| :heavy_multiplication_x: | 4811c206 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/12/) | 2025-06-19 14:36:45 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/12//consoleText) |
| :heavy_multiplication_x: | 4811c206 | [#13](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/13/) | 2025-06-19 14:49:12 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/13//consoleText) |
| :heavy_multiplication_x: | 4811c206 | [#14](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/14/) | 2025-06-19 19:43:55 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/14//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 63dcd6ec | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6687/10/) | 2025-06-20 07:19:16 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250620-071621-63dcd6-pr6687.aar) |
| :heavy_check_mark: | 63dcd6ec | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6687/10/) | 2025-06-20 07:19:57 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250620-071623-63dcd6-pr6687.zip) |
| :heavy_check_mark: | 63dcd6ec | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6687/10/) | 2025-06-20 07:20:13 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250620-071626-63dcd6-pr6687.zip) |
| :heavy_check_mark: | 63dcd6ec | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6687/10/) | 2025-06-20 07:20:15 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250620-071628-63dcd6-pr6687.zip) |
| :heavy_check_mark: | 63dcd6ec | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6687/10/) | 2025-06-20 07:21:04 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250620-071622-63dcd6-pr6687.zip) |
| :heavy_check_mark: | 63dcd6ec | [#10](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6687/10/) | 2025-06-20 07:21:30 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250620-071631-63dcd6-pr6687.zip) |
| :heavy_multiplication_x: | 63dcd6ec | [#15](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/15/) | 2025-06-20 07:26:44 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/15//consoleText) |
| :heavy_check_mark: | 63dcd6ec | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6687/10/) | 2025-06-20 07:28:51 | ~12 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250620-071624-63dcd6-pr6687.zip) |
| :heavy_check_mark: | 63dcd6ec | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/12/) | 2025-06-20 07:44:33 | ~28 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/12//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 68acee76 | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6687/11/) | 2025-06-23 07:16:44 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250623-071319-68acee-pr6687.aar) |
| :heavy_check_mark: | 68acee76 | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6687/11/) | 2025-06-23 07:18:03 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250623-071327-68acee-pr6687.zip) |
| :heavy_check_mark: | 68acee76 | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6687/11/) | 2025-06-23 07:18:50 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250623-071324-68acee-pr6687.zip) |
| :heavy_check_mark: | 68acee76 | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6687/11/) | 2025-06-23 07:18:58 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250623-071322-68acee-pr6687.zip) |
| :heavy_check_mark: | 68acee76 | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6687/11/) | 2025-06-23 07:20:38 | ~7 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250623-071319-68acee-pr6687.zip) |
| :heavy_check_mark: | 68acee76 | [#11](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6687/11/) | 2025-06-23 07:22:49 | ~9 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250623-071331-68acee-pr6687.zip) |
| :heavy_multiplication_x: | 68acee76 | [#16](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/16/) | 2025-06-23 07:24:48 | ~11 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/16//consoleText) |
| :heavy_check_mark: | 68acee76 | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6687/11/) | 2025-06-23 07:30:12 | ~16 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250623-071323-68acee-pr6687.zip) |
| :heavy_multiplication_x: | 68acee76 | [#13](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/13/) | 2025-06-23 07:44:53 | ~31 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/13//consoleText) |
| :heavy_multiplication_x: | 68acee76 | [#17](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/17/) | 2025-06-23 08:27:13 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/17//consoleText) |
| :heavy_multiplication_x: | 68acee76 | [#14](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/14/) | 2025-06-23 08:46:27 | ~26 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/14//consoleText) |
| :heavy_check_mark: | 68acee76 | [#18](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/18/) | 2025-06-23 11:35:54 | ~11 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6687/18//consoleText) |
| :heavy_check_mark: | 68acee76 | [#15](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/15/) | 2025-06-23 11:51:16 | ~27 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6687/15//consoleText) |

saledjenic committed:
refactor(accounts-management)_: rename account selection methods

- Renamed `SelectAccount` to `SetChatAccount`.
- Renamed `SetChatAccount` to `SetChatAccountWithPrivateKey`.
- Updated related test methods to reflect the new naming convention.
- Moved the `LoginParams` struct from the `types` package to `geth_backend` file.
- Adjusted README documentation to reflect changes in account selection methods.

alaibe commented on accounts-management/README.md:
```diff
@@ -0,0 +1,190 @@
+# Accounts Management Package
+
+The Accounts Management package provides a comprehensive solution for Ethereum account creation, storage, and management within the Status application. It handles account generation, keystore operations, and secure storage of private keys.
```
for Status account creation? or at least should not be ethereum as it also does chat. maybe bip44?

saledjenic replied:
Updated.

saledjenic committed:
refactor(accounts-management)_: move KeyStore interface from types package to accountsmanagement package

saledjenic committed:
refactor(accounts-management)_: enhance AccountsManager with persistence interface

- Introduced a new `Persistence` interface to manage account data storage and retrieval.
- Updated `AccountsManager` to require a `Persistence` instance during initialization.
- Refactored related methods to utilize the new persistence layer, improving separation of concerns.
- Updated tests to mock the persistence layer for better isolation and reliability.
- Added error handling for missing persistence and logger instances.

saledjenic committed:
refactor(accounts-management)_: simplify MigrateKeyStoreDir logic

There is no need for the addresses to be provided from outside, the migration should migrate all the files
that are in the keystore dir to another location.

- Removed the addresses parameter from the MigrateKeyStoreDir method in AccountsManager and related interfaces.
- Updated all references and usages of MigrateKeyStoreDir across the codebase to reflect the new signature.
- Adjusted tests and documentation accordingly to ensure consistency and clarity.
